browser/resources/settings/email_aliases_page/styles.ts

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import styled from 'styled-components'
+import { color, spacing } from '@brave/leo/tokens/css/variables'
+import Button from '@brave/leo/react/button'
+
+export const BraveIconCircle = styled.div`
+  align-items: center;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 2.5em;
+  border-radius: 50%;
+  border: #E3E3E8 1px solid;
+  display: flex;
+  min-height: 4.5em;
+  justify-content: center;
+  margin-inline-end: 1.5em;
+  min-width: 4.5em;
+  flex-grow: 0;
+`
+
+export const BraveIconWrapper = styled.div`
+  transform scale(2);
+  display: inline-block;
+`
+
+export const Card = styled.div`
+  background-color: ${color.container.background};
+  border: none;
+  overflow: hidden;
+  padding: ${spacing.l} ;
+`
+
+export const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`
+
+export const Col = styled.div`
+  display: flex;
+  flex-direction: column;
+`
```

Comment:
There's also a `Flex` component in `$web-common/Flex` if you want to use it - its like these but much more flexible

Reply:
Leaving as-is for now, but I will consider replacing in the future